### ISSUE TITLE
Use case statement for crash enum

### ISSUE THREAD

**Issue by ghost** [davidker/unisys#109]
[KanBoard-24525](http://ustr-erl-6690.na.uis.unisys.com/?controller=task&action=show&task_id=24525&project_id=37)

``` 
 > 		if (visorchannel_write(controlvm_channel,
 > 				       local_crash_msg_offset,
 > 				       msg,
 > 				       sizeof(struct controlvm_message)) < 0) {
 > 			POSTCODE_LINUX_2(SAVE_MSG_BUS_FAILURE_PC,
 > 					 POSTCODE_SEVERITY_ERR);
 > 			return;
 > 		}
 > 	} else {
 > 		local_crash_msg_offset += sizeof(struct controlvm_message);
 > 		if (visorchannel_write(controlvm_channel,
 > 				       local_crash_msg_offset,
 > 				       msg,
 > 				       sizeof(struct controlvm_message)) < 0) {
 > 			POSTCODE_LINUX_2(SAVE_MSG_DEV_FAILURE_PC,
 > 					 POSTCODE_SEVERITY_ERR);
 > 			return;
 > 		}
 ```
 So an enum yet you don't check the other value?  case statement perhaps?

**Comment by ghost** [comment]
**Branch:** githubissue-109-upstream-next-TurningWrenches
**Commit(s):** 
2a04c5403b78abeff8c1e1c891f6db7aeea5a83f
**checkpatch-cleared:** Yes
**T710-W1 verified:** Yes, but also pending response from e-mail "Partition Desktop Behavior"

Update: No longer waiting on this e-mail. Issue presents in 3.10 kernel.

**Comment by davidker** [comment]
Was this built off of for-later?

**Comment by ghost** [comment]
That should say "upstream-next". Sorry about the typo :(

**UPDATE:** Fixed.

**Comment by davidker** [comment]
Closed committed at 36309d3b105c650eb0398afd1675df30fb570e5d
